kubernetes/zenko/charts/burry/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for elasticsearch-curator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+cronjob:
+  suspend: true
+  # Every 5 minutes
+  schedule: "5 * * * *"
+  annotations: {}
+
+pod:
+  annotations: {}
+
+
+# serviceAccount:
```

Comment:
There's no templating attached to these.

Reply:
Removed all unnecessary options